nx-X11/programs/Xserver/include/servermd.h

```diff
@@ -48,308 +48,19 @@ SOFTWARE.
-
-#endif /* macII */
-
-#if (defined(mips) || defined(__mips))
-
-#if defined(MIPSEL) || defined(__MIPSEL__)
-# define IMAGE_BYTE_ORDER	LSBFirst        /* Values for the PMAX only */
-# define BITMAP_BIT_ORDER	LSBFirst
-# define GLYPHPADBYTES		4
-#else
-# define IMAGE_BYTE_ORDER	MSBFirst        /* Values for the MIPS only */
+#elif X_BYTE_ORDER == X_BIG_ENDIAN
+# define IMAGE_BYTE_ORDER	MSBFirst
 # define BITMAP_BIT_ORDER	MSBFirst
-# define GLYPHPADBYTES		4
-#endif
-
-#endif /* mips */
-
-#if defined(__alpha) || defined(__alpha__) || defined(__alphaCross)
-# define IMAGE_BYTE_ORDER	LSBFirst	/* Values for the Alpha only */
-
-# if defined(XF86MONOVGA) || defined(XF86VGA16) || defined(XF86MONO)
-#  define BITMAP_BIT_ORDER      MSBFirst
-# else
-#  define BITMAP_BIT_ORDER      LSBFirst
-# endif
-
-# if defined(XF86MONOVGA) || defined(XF86VGA16)
-#  define BITMAP_SCANLINE_UNIT  8
-# endif
-
-# define GLYPHPADBYTES		4
-
-#endif /* alpha */
-
-#if defined (linux) && defined (__s390__)
-
-#define IMAGE_BYTE_ORDER      	MSBFirst
-#define BITMAP_BIT_ORDER      	MSBFirst
-#define GLYPHPADBYTES         	4
-
-#define BITMAP_SCANLINE_UNIT	8
-
-#endif /* linux/s390 */
-
-#if defined (linux) && defined (__s390x__)
-
-#define IMAGE_BYTE_ORDER       MSBFirst
-#define BITMAP_BIT_ORDER       MSBFirst
-#define GLYPHPADBYTES          4
-
-#define BITMAP_SCANLINE_UNIT	8
-
-#endif /* linux/s390x */
-
-
-#if defined(__ia64__) || defined(ia64)
-# define IMAGE_BYTE_ORDER	LSBFirst
-
-# if defined(XF86MONOVGA) || defined(XF86VGA16) || defined(XF86MONO)
-#  define BITMAP_BIT_ORDER      MSBFirst
-# else
-#  define BITMAP_BIT_ORDER      LSBFirst
-# endif
-
-# if defined(XF86MONOVGA) || defined(XF86VGA16)
-#  define BITMAP_SCANLINE_UNIT  8
-# endif
-
-# define GLYPHPADBYTES		4
-
-#endif /* ia64 */
-
-#if defined(__amd64__) || defined(amd64) || defined(__amd64)
-# define IMAGE_BYTE_ORDER	LSBFirst
-
-# if defined(XF86MONOVGA) || defined(XF86VGA16) || defined(XF86MONO)
-#  define BITMAP_BIT_ORDER      MSBFirst
-# else
-#  define BITMAP_BIT_ORDER      LSBFirst
-# endif
-
-# if defined(XF86MONOVGA) || defined(XF86VGA16)
-#  define BITMAP_SCANLINE_UNIT  8
-# endif
-
-# define GLYPHPADBYTES		4
-/* ???? */
-#endif /* AMD64 */
-
-#ifdef stellar
-
-#define IMAGE_BYTE_ORDER	MSBFirst       /* Values for the stellar only*/
-#define BITMAP_BIT_ORDER	MSBFirst
-#define	GLYPHPADBYTES		4
-#define IMAGE_BUFSIZE		(64*1024)
-/*
- * Use SysV random number generator.
- */
-#define random rand
-
-#endif /* stellar */
-
-#ifdef luna
-
-#define IMAGE_BYTE_ORDER        MSBFirst   	/* Values for the OMRON only*/
-#define BITMAP_BIT_ORDER	MSBFirst
-#define	GLYPHPADBYTES		4
-
-#ifndef mc68000
-#endif
-
-#endif /* luna */
-
-#if	(defined(SVR4) && defined(i386)) || \
-	defined(__alpha__) || defined(__alpha) || \
-	defined(__i386__) || defined(__i386) || \
-	defined(__s390x__) || defined(__s390__)
-  
-#ifndef IMAGE_BYTE_ORDER
-#define IMAGE_BYTE_ORDER	LSBFirst
-#endif
-
-#ifndef BITMAP_BIT_ORDER
```

Comment:
What about this one? I guess we should mention these commits as well, even though sometimes changes were just cherry-picked (contrary to fully applied.)